docker-compose.yml

```diff
@@ -17,11 +16,12 @@ services:
     depends_on:
       postgres:
         condition: service_healthy
+      solr:
```

Comment:
These changes actually got made in the search PR, which hasn't been merged yet. They really improve the developer experience, so I added them here, as well. Not optimal, but functional.